core/src/main/scala/org/apache/spark/scheduler/ExecutorLossReason.scala

```diff
@@ -70,7 +71,7 @@ case class ExecutorProcessLost(
  * This is used by the task scheduler to remove state associated with the executor, but
  * not yet fail any tasks that were running in the executor before the executor is "fully" lost.
  *
- * @param hostOpt it will be set by [[TaskSchedulerImpl]] when the host is decommissioned too
+ * @param workerHost it's defined when the worker is decommissioned too
```

Comment:
nit: "it's" -> "it is"

Also, should we explicitly bring out the word 'host' here ? "It is defined when the worker _host_ is decommissioned too"

Reply:
I think "worker" should be enough.